concepts/functions/about.md

```diff
@@ -19,6 +22,55 @@ func PrintHelloName(name string) {
 }
 ```
 
+Parameters of the same type can be declared together, followed by a single type declaration.
+
+```go
+import "fmt"
+
+func PrintGreetingName(greeting, name string) {
+  fmt.Println(greeting + " " + name)
+}
+```
+
+## Parameters vs. Arguments
+
+Let's quickly cover two terms that are often confused together: `parameters` and `arguments`. 
+Function parameters are the names defined in the function's signature, such as `greeting` and `name` in the function `PrintGreetingName` above.
+Function arguments are the concrete values passed to the function parameters when we invoke the function.
+For instance, in the example below, `"Hello"` and `"Katrina"` are the arguments passed to the `greeting` and `name` parameters:
+
+```go
+PrintGreetingName("Hello", "Katrina")
+```
+
+## Pass by Value & Pass by Pointer
+
+It is also important to clarify the concept of passing by value as well as passing by pointer (sometimes referred to as _by reference_).
```

Comment:
```suggestion
## Passing by value vs passing by reference

It is also important to clarify the concept of passing by value as well as passing by reference.
```

The term is "passing by reference". We can clarify in the text that reference here means passing a pointer (you are already doing that though).

Reply:
I think this is related to the other comment you made: There's only pass by value in Go so I chose to say "pass by pointer", which is technically correct but makes the concept a bit more complex. Happy to change it to simplify it a bit :)